### PR TITLE
[produce,deliver,spaceship] Support for new AppStore Connect languages: Catalan, Croatian, Czech, Hindi, Hungarian, Polish, Romanian, Slovak, and Ukrainian

### LINKED ISSUE #13938: New App Store Connect languages
image

Links to https://help.apple.com/app-store-connect/#/deve6f78a8e2

### PR DESCRIPTION
**Pull request by cleanbit**
* New languages:
	Catalan, Croatian, Czech, Hindi, Hungarian, Polish, Romanian, Slovak, and Ukrainian

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

New App Store Connect languages were added some time ago and fastlane doesn't support them yet.

### Description

Added new language constants and language mappings

---
close #13938

### CODE REVIEW

**Comment by janpio** [comment]
Awesome @Zymantas! Where did you get/extract the language codes to add to your files? Anything else worth writing down for future updates of these files?

**Comment by Zymantas** [comment]
I've taken language codes from the Xcode project localisations (where you add/remove translated languages of your app) and put them here. As I hoped, it worked on both platforms - iOS and macOS! :)

I didn't find anything that's missing with these new languages, but I'll be retesting again when real-world translations will be ready (sometime next week as our translators promised). I'll let you know if this won't be merged till then.

File: spaceship/lib/assets/languageMapping.json [hunk 1]
```diff
@@ -11,6 +17,12 @@
         "game-center": true,
         "itc_locale": "zh-Hant"
     },
+    {
+        "locale": "cs-CZ",
+        "name": "Czech",
+        "game-center": false,
+        "itc_locale": "cz"
```
Comment (joshdholtz) [inline spaceship/lib/assets/languageMapping.json]: Should this be `cs`? 🤔

**Review by joshdholtz — CHANGES_REQUESTED** [review]
One small change request! I think 😇

**Comment by joshdholtz** [comment]
@janpio This can also be achieve by running `Spaceship::Tunes.client.available_languages.sort` (for getting the language codes at least) 💪 🚀

File: spaceship/lib/assets/languageMapping.json (hunk 1 above)
Comment (Zymantas) [reply spaceship/lib/assets/languageMapping.json]: Ahhhh, you're correct.

**Comment by janpio** [comment]
Do we maybe also want to add comments that "interlink" these 3 files/locations? Because if you update one of them, you should always update the others, correct? Can we also add some information in a comment where the other data comes from?

**Comment by joshdholtz** [comment]
Aye aye! Can do can do 💪

**Comment by joshdholtz** [comment]
@janpio Updated with more info and link to this PR as an example 😊

**Comment by janpio** [comment]
Ok, I fixed the comment a bit and added a similar one to the second file.

What about the JSON - we can't just add a comment there :/ What do we do there?

But ohoh, I found another one: https://github.com/fastlane/fastlane/blob/master/spaceship/lib/assets/languageMappingReadable.json

**Comment by joshdholtz** [comment]
@janpio I did some searching through there and it doens't look like https://github.com/fastlane/fastlane/blob/master/spaceship/lib/assets/languageMappingReadable.json actually gets used anywhere. There are two methods that reference it (`from_itc_readable_to_itc` and `from_itc_to_itc_readable`) but these actually don't get referenced anywhere either. I think its okay to not edit this language file and probably also okay to remove that file and those methods in a future PR? 🤔

Also going to ping @KrauseFx since he add this file back in 2015 😉 - https://github.com/fastlane-old/spaceship/commit/dd510037d377bff7efa9b8b5b7d3f5bc44f88672#diff-355ef28abf38992915b1a2e1724703eb

**Comment by janpio** [comment]
Deleteable code is best code ❌ 💀

**Review by joshdholtz — APPROVED** [review]

**Comment by joshdholtz** [comment]
Let's ship this 💪

File: spaceship/lib/assets/languageMapping.json [hunk 2]
```diff
@@ -201,6 +243,12 @@
             "es"
         ]
     },
+    {
+        "locale": "sk-SK",
+        "name": "Romanian",
```
Comment (max-ott) [inline spaceship/lib/assets/languageMapping.json]: @joshdholtz This should be `Slovak`

**Comment by janpio** [comment]
Ohoh. Can you please create another PR? Thanks.

**Comment by Zymantas** [comment]
Ohhh :(. Since this is my PR, I'll create new one with a fix.

**Pull request merged**